### PR TITLE
Update 2020-05-04_11-00-00_mallet-end-to-end-en.md

### PR DESCRIPTION
**Pull request by olgahryniuk**
Deleting solcjs installation instructions, replacing solcjs reference with `solc`, adding a different command instead (provided by James Browning).

### CODE REVIEW

File: resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md [hunk 1]
```diff
@@ -164,14 +135,12 @@ If the file was correctly compiled, there should be a `.bin` file in your direct
 
     _myContract_sol_HelloWorld.bin
```
Comment (james-iohk) [inline resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md:136]: Change filename to `HelloWorld.bin`

File: resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md [hunk 2]
```diff
@@ -150,10 +120,11 @@ Create a `myContract.sol` file:
     }
     EOF
 
+**2. Compile with `solc`:**
 
-**2. Compile with `solcjs`:**
+To compile with `solc`, you will need to use the Docker command. For this, first install the [Docker Engine](https://docs.docker.com/engine/install/), then run:
 
-    solcjs --bin --abi --base-path . ./myContract.sol
+    $ docker run -v $(pwd):/sources ethereum/solc:0.5.1 -o /sources --bin --abi /sources/myContract.sol
```
Comment (james-iohk) [inline resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md:127]: `$` at the start is correct to indicate this is a shell command, but it hasn't been used for other commands in this page so I'd suggest either:   remove it here OR add it before all other shell commands.

Comment (james-iohk) [inline resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md:127]: Add `--rm`:
`docker run --rm -v $(pwd):/sources ethereum/solc:0.5.1 -o /sources --bin --abi /sources/myContract.sol`

Comment (olgahryniuk) [reply resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md]: removed `$` here

File: resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md (hunk 1 above)
Comment (olgahryniuk) [reply resources/content/articles/en/2020-05-04_11-00-00_mallet-end-to-end-en.md]: renamed

**Review by james-iohk — CHANGES_REQUESTED** [review]
Please make the following changes:
- Line 234: change `solcjs` to `solc`
-  Lines 217 & 229: change all occurrences of `_myContract_HelloWorld.bin` to `HelloWorld.bin`

**Review by james-iohk — APPROVED** [review]

**Review by james-iohk — APPROVED** [review]

**Pull request merged**